scripts/eval_lid_curve.py

```
@@ -0,0 +1,374 @@
+    lines.append("## 結論\n")
+    per_cond_lang_cross = {}   # cond -> {lang: crossing length or None}
+    per_cond_overall_single = {}   # cond -> {length: overall single-LID acc}
+    per_cond_agree = {}        # cond -> {length: (agree_rate, agree_acc)}
+
+    for cond in conditions:
+        lines.append(f"### {cond}\n")
+        rows_by_length_overall = {L: [r for r in rows if r["condition"] == cond and r["length"] == L]
+                                   for L in BIN_SECONDS}
+        overall_cross = find_95_crossing(rows_by_length_overall, "wtiny_acc")
+        lines.append(
+            f"- 単独LID（whisper-tiny）が全体で95%を超え、それ以降も維持する最短長: "
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)